ci/ReleaseTool/Common.cs

```diff
@@ -135,6 +200,81 @@ public static bool UpdateVersionFile(GitClient gitClient, string fileContents, s
             return true;
         }
 
+        public static bool UpdateUnrealEngineVersionFile(GitClient client, List<string> versions, string unrealEngineVersionFile)
```

Comment:
Mostly the same as before but with consistent error handling and a return value for if anything has changed.